pkg/logger/api.go

```diff
@@ -54,3 +54,8 @@ func (l ApiLogger) Warningf(format string, a ...interface{}) {
 func (l ApiLogger) Failuref(format string, a ...interface{}) {
 	l.logger.Info(fmt.Sprintf(format, a...))
 }
+
+func (l ApiLogger) Write(p []byte) (n int, err error) {
+	// Not implemented yet
```

Comment:
Not OK, we need to implement this. I realize I set a bad example on the `KubeHTTP` library, but there were specific reasons why I left that unimplemented.

The implementation for this can be very simple:

```golang
l.logger.Info(string(p))
```